Content.Server/Ame/Components/AmeControllerComponent.cs

```diff
@@ -72,10 +72,23 @@ public sealed partial class AmeControllerComponent : SharedAmeControllerComponen
     [DataField("nextUpdate")]
     public TimeSpan NextUpdate = default!;
 
+    /// <summary>
+    /// The next time this will try to update the controller UI.
+    /// </summary>
+    [DataField("nextUIUpdate")]
+    public TimeSpan NextUIUpdate = default!;
+
     /// <summary>
     /// The the amount of time that passes between injection attempts.
     /// </summary>
     [DataField("updatePeriod")]
     [ViewVariables(VVAccess.ReadWrite)]
     public TimeSpan UpdatePeriod = TimeSpan.FromSeconds(10.0);
+
+    /// <summary>
+    /// The maximum amount of time that passes between UI updates.
+    /// </summary>
+    [DataField("updateUIPeriod")]
```

Comment:
```suggestion
    [DataField]
```

Reply:
since this does not need to be referenced anywhere  I just removed the DataField part